ui/page/wallets/restore_page.go

```diff
@@ -656,50 +656,80 @@ func (pg *Restore) HandleUserInteractions() {
 	}
 }
 
-// HandleKeyEvent is called when a key is pressed on the current window.
+// KeysToHandle returns an expression that describes a set of key combinations
+// that this page wishes to capture. The HandleKeyPress() method will only be
+// called when any of these key combinations is pressed.
 // Satisfies the load.KeyEventHandler interface for receiving key events.
-func (pg *Restore) HandleKeyEvent(evt *key.Event) {
+func (pg *Restore) KeysToHandle() key.Set {
+	if pg.isRestoring {
+		return "" // don't capture keys while restoring, problematic?
+	}
+	// TODO: Once user starts editing any of the input boxes, the arrow up, down
+	// and enter key signals are no longer received.
```

Comment:
Only issue left: no idea how to fix yet, seems the editors from the upstream package no longer propagate arrow and enter key events to the caller even if requested.